src/Microsoft.ML.Transforms/KeyToVectorMapping.cs

```diff
@@ -478,7 +480,7 @@ public override SchemaShape GetOutputSchema(SchemaShape inputSchema)
             {
                 if (!inputSchema.TryFindColumn(colInfo.Input, out var col))
                     throw Host.ExceptSchemaMismatch(nameof(inputSchema), "input", colInfo.Input);
-                if ((col.ItemType.ItemType.RawKind == default) || !(col.ItemType.IsVector || col.ItemType is PrimitiveType))
+                if ((col.ItemType.GetItemType().RawKind == default) || !(col.ItemType is VectorType || col.ItemType is PrimitiveType))
                     throw Host.ExceptSchemaMismatch(nameof(inputSchema), "input", colInfo.Input);
 
                 var metadata = new List<SchemaShape.Column>();
```

Comment:
Ugh. I thought I was getting close to done and then I see no, github "abbreviated" the remaining files which I have to open one at a time. La ti da... 😛 😆